lib/geoblacklight/wms_layer.rb

```diff
@@ -1,6 +1,7 @@
 module Geoblacklight
   class WmsLayer
     def initialize(params)
+      params = params.to_unsafe_h unless params.is_a?(Hash)
       @params = params.to_h.merge(Settings.WMS_PARAMS)
```

Comment:
Good point. We should explicitly permit the params. I'll move this into the WMS controller.